packages/avatars/src/elements/Avatar.tsx

```diff
@@ -26,15 +30,15 @@ const AvatarComponent = forwardRef<HTMLElement, IAvatarProps>(
     },
     ref
   ) => {
-    const computedStatus = badge === undefined ? status : 'active';
+    const computedBadge = ['string', 'number'].includes(typeof badge) ? `${badge}` : null;
+    const computedStatus = computedBadge ? 'active' : status;
```

Comment:
Should this be reverted?

Reply:
@Francois-Esquire I'm a bit confused here with the use of icons and sizes, so please explain so I can understand the implementation better. I might be missing a technical explanation but please if the following is incorrect, educate away!

It looks like you're using the `arrow-left` existing Garden 12px icon for the `transfer only` icon at both sizes? I remember walking you through the designs early on and [also stated here](https://github.com/zendeskgarden/react-components/pull/1358#issuecomment-1143114025) that that isn't optimal. We need two separate icon sizes used - a 12px variant and a 16px variant, as the grids are different when designed. The same applies for the `clock` icon. Scaling a 12px icon to 16px will cause off-pixel alignment and appear blurry on lower res screens. This why we have 12px and 16px icon variants in the first place.

We have the `clock` icon inside the current Garden icon library but not the specifically designed `transfers only` icon which has subtle differences from the Garden `arrow-left` icon.